packages/kmod/kmod.spec

```diff
@@ -30,61 +33,52 @@ cp COPYING COPYING.LGPL
 cp tools/COPYING COPYING.GPL
 
 %build
-
-%define _configure ../configure
-
-mkdir static-build
-pushd static-build
-%cross_configure \
-  --with-noarch-pkgconfigdir=%{_cross_pkgconfigdir} \
-  --with-zlib \
-  --with-zstd \
-  --without-openssl \
-  --disable-manpages
-
-%make_build LDFLAGS="-all-static"
-
-popd
-
+CONFIGURE_OPTS=(
+  -Dzlib=enabled
+  -Dzstd=enabled
+  -Dopenssl=disabled
+  -Dmanpages=false
+  -Dxz=disabled
+  -Dfishcompletiondir=no
+  -Dbashcompletiondir=no
+  -Dzshcompletiondir=no
+)
+
+%global _cross_vpath_builddir dynamic-build
 mkdir dynamic-build
-pushd dynamic-build
+%cross_meson "${CONFIGURE_OPTS[@]}"
+%cross_meson_build
 
-%cross_configure \
-  --with-noarch-pkgconfigdir=%{_cross_pkgconfigdir} \
-  --with-zlib \
-  --with-zstd \
-  --without-openssl \
-  --disable-manpages
-
-%make_build
-
-popd
+%global _cross_vpath_builddir static-build
+mkdir static-build
+%{cross_meson} \
+  --default-library=static \
+  --prefer-static \
+  "${CONFIGURE_OPTS[@]}"
+export LDFLAGS="-all-static"
+%cross_meson_build
 
 %install
-pushd dynamic-build
-%make_install
-popd
+%global _cross_vpath_builddir dynamic-build
+%cross_meson_install
 
 pushd static-build
-install -p tools/kmod %{buildroot}%{_cross_bindir}
-
-install -d %{buildroot}%{_cross_sbindir}
-ln -s ../bin/kmod %{buildroot}%{_cross_sbindir}/modprobe
+install -d %{buildroot}%{_cross_bindir}
+ln -s kmod %{buildroot}%{_cross_bindir}/modprobe
```

Comment:
This doesn't seem to be installing the static build, it's just creating a symlink to `kmod` from the previous install.